packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -438,6 +428,17 @@ export async function createMember(
             true,
             datasetProvider
         );
+
+        // Refresh corresponding tree parent to reflect addition
+        const findCorrespondingNode = contextually.isFavorite(parent)
+            ? datasetProvider.findNonFavoritedNode.bind(datasetProvider)
+            : datasetProvider.findFavoritedNode.bind(datasetProvider);
```

Comment:
Can we call the `findEquivalentNode` method here to reduce duplication?

Reply:
Yep! Not sure how I missed that, I added the wrapper function but didn't update the logic to use it 🤦‍♂️